core/templates/dev/head/galleries/Gallery.js

```diff
@@ -218,7 +218,7 @@ oppia.controller('Gallery', [
     });
 
     // Below is the width of each tile.
-    var TILE_DISPLAY_WIDTH = 211.953;
+    var TILE_DISPLAY_WIDTH = 208;
```

Comment:
Does this tie in to some CSS value somewhere? If so, would be nice to reference it in the comment.